homework/tests/example.spec.js

```diff
@@ -17,15 +17,115 @@ const data = [
 
```

Comment:
All the selectors in the code could described as constants if they do repeat. For example:

const firstNumberInputField = '#number1Field';

So then you can use it in code like if (await page.locator(firstNumberInputField ).isVisible())

Image situation if you have 2000 tests where selector is written in a way like you did, and it changes to something like '#numberXField' - now you would need to refactor code and change it in every single place instead of easily updating the selector constant.